provisioning/roles/postgres/templates/srv/postgres/initdb/02_init_ewallet_users.sql.j2

```diff
@@ -0,0 +1,3 @@
+# Create ewallet users and give CREATEDB permission so that mix ecto.drop & mix ecto.create works
+CREATE USER ewallet WITH PASSWORD 'ewallet_weak_password' CREATEDB;
```

Comment:
`WITH PASSWORD` will be deprecated in future PostgreSQL version; can you change this to `WITH ENCRYPTED PASSWORD`?